README.md

```diff
@@ -92,6 +92,27 @@ For `eglot`, use:
 (add-to-list 'eglot-server-programs '(elixir-mode "path-to-elixir-ls/release/language_server.sh"))
 ```
 
+</details>
+<details>
+  <summary>Helix Installation Instructions</summary>
+
+From : http://elixirforum.com/t/helix-editor-for-elixir-development/54964
+
+Put the following configuration in .config/helix/languages.toml, and happy coding!
+
+```toml
+[[language]]
+name = "elixir"
+scope = "source.elixir"
+injection-regex = "elixir"
+file-types = ["ex", "exs"]
+roots = ["mix.exs"]
+auto-format = true
+diagnostic-severity = "Hint"
+comment-token = "#"
+indent = {tab-width = 2, unit = " "}
+language-server = {command = "elixir-ls"}
+```
```

Comment:
Please add some example config, basing on https://elixirforum.com/t/helix-editor-for-elixir-development/54964/24?u=lukaszsamson it's done like that
```
config = { elixirLS.dialyzerEnabled = false }
```

Reply:
Sure. I'll test it first and modify the PR once that's done.